addons/edi/tests/common.py

```diff
@@ -204,3 +204,19 @@ def _execute_edi_for(self, document_type):
         doc = EdiDocument.create({'doc_type_id': document_type.id})
         doc.action_execute()
         return doc
+
+    def _prepare_edi_for(self, document_type):
```

Comment:
I'm not sure I understand the need for this method apart from being an helper function. Can you try explaining again what are you trying to achieve? What's the difference between calling these methods from here instead of directly in the tests?

Reply:
_execute_edi_for which is some lines above same file is thought and used like a generic helper that is used to create manually EDI and execute it.

There are some EDI unit tests that are not developed yet. In my story was needed to prepare first and to change the prepare date because the prepare date is in the name of the file. To be statically i need to update the prepare_date before execution which cannot be done in existing helper _execute_edi_for. Of course could be done separately in unit test, but thought to be consistent with _execute_edi_for as we may need it for other EDI unit tests too.
Hope it makes sense

Reply:
I see, it makes sense, it doesn't hurt to have it. What about the CRON execution comment in the dosctring? I'm not sure if it's relevant.